### ISSUE TITLE
(Bug report) On macOS, forwardInNoteHistory and backInNoteHistory shortcuts clobber system functionality

### ISSUE THREAD

**Issue by yllekz** [TriliumNext/Trilium#3708]
### Trilium Version

0.59.1

### What operating system are you using?

macOS

### What is your setup?

Server access only

### Operating System Version

macOS Ventura 13.2.1

### Description

On macOS, a standard system convention to move a cursor between whole words is "Option (alt) + left/right arrow key" that of which I regularly use to quickly shift the cursor through typed words. In Trilium, it seems this is mapped to backInNoteHistory and forwardInNoteHistory which overrules the system. My muscle memory is causing me to invoke this which ends up sending me to different Trilium notes pages and disrupting my workflow.

Is it possible to disable this hotkey by default or move it to a different one for macOS? I know it can be fixed in options, but the default behavior should respect the system, in my opinion.

**Comment by zadam** [comment]
Changed to CMD+Left/Right which seems to be the standard on Mac.

**Comment by jjkoehorst** [comment]
Can this be reopened? CMD + Left / CMD + Right are reserved to jump to the beginning of the line / end of the line. I use this a lot! 

I disabled this in the shortcut settings but every time I do CMD + Left it jumps to a previous tab? Not sure if this is part of the shortcuts as I was unable to see another CMD + Left option...

**Comment by eliandoran** [comment]
@jjkoehorst ,

The OP mentioned that Opt + Left/Right are the convention. So which one is it?

> On macOS, a standard system convention to move a cursor between whole words is "Option (alt) + left/right arrow key"

**Comment by jjkoehorst** [comment]
@eliandoran 

> [@jjkoehorst](https://github.com/jjkoehorst) ,
> 
> The OP mentioned that Opt + Left/Right are the convention. So which one is it?
> 
> > On macOS, a standard system convention to move a cursor between whole words is "Option (alt) + left/right arrow key"

True, between words, CMD + Left / Right is beginning of the line / end of the line.

**Comment by eliandoran** [comment]
New default shortcuts for history on Mac will be:

- ⌘ + [ → Back
- ⌘ + ] → Forward

This will not affect existing installations/documents, only new ones.
